concept/thing/attribute.feature

```diff
@@ -43,7 +43,7 @@ Feature: Concept Attribute
     Given attribute(email) set annotation: @regex("\S+@\S+\.\S+")
     Given transaction commits
     Given connection open write transaction for database: typedb
-    Given set time zone: Europe/London
+    Given set time-zone: Europe/London
```

Comment:
We had two different types of these steps, I inclined to this one following [this](https://english.stackexchange.com/questions/3934/time-zone-vs-timezone)

Will require server's update when the ref is updated on the server side! Don't be scared